std_msgs/Header header

# center (m, m)
float32 x
float32 y
# dims (m, m)
float32 w
float32 h
# ccw angle (rad)
float32 a
